dashboard/src/App.tsx

```diff
@@ -31,24 +28,9 @@ export const Parrot: FC = () => {
             gap: 1,
           }}
         >
-          <Header />
-
-          <Box
-            sx={{
-              display: "flex",
-              mb: 1,
-              gap: 1,
-              flexDirection: { xs: "column", sm: "row" },
-              alignItems: { xs: "start", sm: "center" },
-              flexWrap: "wrap",
-              justifyContent: "space-between",
-            }}
-          >
-            <Typography level="h2" component="h1">
-              Projects
-            </Typography>
-          </Box>
-          <ProjectsOverview />
+          <BrowserRouter>
+            <Routes />
+          </BrowserRouter>
         </Box>
       </Box>
     </CssVarsProvider>
```

Comment:
I think you should consider moving the Box element out somewhere else so this App.tsx file look a bit cleaner.

Reply:
moved to main.tsx